### ISSUE TITLE
Renaming Templates

### ISSUE THREAD

**Issue by allanabiud** [comictagger/comictagger#576]
I'm requesting for a new feature: renaming templates.

So one can have different renaming formats for issues, Trade Paperbacks, Hardcovers etc. and have it be easy to apply them.

**Comment by lordwelch** [comment]
So you want to have a list of templates you can choose from?

**Comment by allanabiud** [comment]
Exactly. And even be able to create templates for the different formats.

**Comment by frijolie** [comment]
another thought along these same lines, while the template selection is a good idea, i think it would also be cool if it could be more smart/dynamic. Say the proper template could be applied based on the format in metadata (TPB, Series, Limited Series, One Shot, etc). So you'd define a renaming template for each format you want to apply. Then, after tagging the file with proper metadata, the respective template would be populated by the value in format when you trigger a file rename.
